### PR TITLE
feat: LT-1143 - Valid control list entry

### PR DESCRIPTION
**Pull request by jfaracik**

### CODE REVIEW

File: goods/forms.py [hunk 1]
```diff
@@ -1,51 +1,59 @@
...
-                 classes=['govuk-radios--inline']),
-    TextInput(title='Part Number',
-              name='part_number',
-              optional=True),
-])
+
+def add_goods_questions():
+    return Form(title='Add Good',
+                questions=[
+                    TextArea(title='Description of good',
```
Comment (BenAndrew-Kainos) [inline goods/forms.py]: Might want to move these to strings (if we're still using it after a discussion later)

File: goods/forms.py [hunk 2]
```diff
@@ -170,5 +179,6 @@ def delete_good_form(good):
                 buttons=[
                     Button(value='Yes, delete the good', action='submit', style=ButtonStyle.WARNING),
                     Button(value='Cancel', action='', style=ButtonStyle.SECONDARY, link=reverse_lazy('goods:edit',
-                                                                                                     kwargs={'pk': good['id']}))
+                                                                                                     kwargs={'pk': good[
+                                                                                                         'id']}))
```
Comment (BenAndrew-Kainos) [inline goods/forms.py]: Not sure about this indentation. Maybe just split the Button in the middle after the style argument

File: goodstype/forms.py [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
-                            value='No')
-                 ],
-                 classes=['govuk-radios--inline'])
-])
+from core.services import get_control_list_entries
+
+
+def goods_type_form():
+    return Form(title='Add a description', questions=[
+        TextArea(title='Give a short description of your goods.',
```
Comment (BenAndrew-Kainos) [inline goodstype/forms.py]: Same issue as above with strings

File: ui_automation_tests/fixtures/add_an_application.py [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+
+from helpers.utils import Timer, get_lite_client
 
 
-@fixture(scope="module")
+@fixture(scope='module')
 def add_an_application(driver, request, api_url, exporter_url, context):
     timer = Timer()
-    api = get_or_create_attr(context, 'api', lambda: SeedData(api_url=api_url, logging=True))
+    lite_client = get_lite_client(context)
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/fixtures/add_an_application.py]: nice 👍

File: ui_automation_tests/fixtures/add_goods.py [hunk 5]
```diff
@@ -2,34 +2,33 @@
...
     overview_page = ApplicationOverviewPage(driver)
-    api = get_or_create_attr(context, 'api', lambda: SeedData(api_url=api_url, logging=False))
-    api.add_good_end_product_false()
-    context.goods_name = api.context['goods_name']
+    lite_client = get_lite_client(context)
+    lite_client.add_good_end_product_false()
+    context.goods_name = lite_client.context['goods_name']
     driver.get(url)
-    driver.execute_script("document.getElementById('goods').scrollIntoView(true);")
+    driver.execute_script('document.getElementById(\'goods\').scrollIntoView(true);')
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/fixtures/add_goods.py]: We have a helper function for this called scroll_into_element_by_id

Comment (jfaracik) [reply ui_automation_tests/fixtures/add_goods.py]: It doesn't exist in exporter

Comment (BenAndrew-Kainos) [reply ui_automation_tests/fixtures/add_goods.py]: it does - helpers/helpers.py

**Pull request merged**